pages/articles/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { ArticlesComponent } from '../../components/ArticlesComponent/ArticlesComponent';
+
+export default function Articles() {
+  return (<>
+    <h1>Articole</h1>
+    <ArticlesComponent></ArticlesComponent>
```

Comment:
Ar trebui să folosești Self-Closing-Tag.

```suggestion
    <ArticlesComponent />
```